### PR TITLE
Ward nights!

### PR DESCRIPTION
**Pull request by jim**
I am opening this PR to track my progress toward completing this task.

- [x] Fetch data from Google sheet in JSON format.
    Note that a Google API key with the Sheets API enabled is needed to request data from [this URL](https://sheets.googleapis.com/v4/spreadsheets/1xnt4kZI9Ruinw91wM-nnWftsFD-ZaKaozepdNXeIrpo/values/A1%3AJ100?key=GOOGLE_API_KEY).
- [x] Load JSON with spider.
- [x] Modify structure/format of spreadsheet to simplify programmatic consumption.
- [x] Generate a list of dates based on descriptions such as  "third Thursday of the month"
- [x] Generate events using JSON data.
- [x] Decide how to handle Google API token in prod (env variable?). _Using an environment variable._

### CODE REVIEW

**Comment by jim** [comment]
Data fetched from Google sheet is looking good. Working on generating the events now.

**Comment by jim** [comment]
@r-wei I need to do a little more manual testing, but I think this PR is ready for a code review.

File: documenters_aggregator/spiders/ward_night.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+        Generate ID. We are assuming that there will only be a single event in each
+        ward in a single day.
+        """
+
+        values = {
+            'ward': row[Row.WARD],
+            'date': date.strftime('%Y-%m-%d')
+        }
+        id = 'ward{ward}-{date}'.format(**values)
+        return id
```
Comment (r-wei) [inline documenters_aggregator/spiders/ward_night.py]: Are these the standardized ids?

Comment (jim) [reply documenters_aggregator/spiders/ward_night.py]: No, they aren't! This predates that PR, but I will make that change. Good catch.

File: documenters_aggregator/spiders/ward_night.py [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    name = 'ward_night'
+    allowed_domains = ['sheets.googleapis.com/v4/']
+    start_urls = []  # assigned in __init__
+    start_date = datetime.today()
+
+    def __init__(self, google_api_key=GOOGLE_API_KEY, spreadsheet_url=SPREADSHEET_URL,
+                 start_date=datetime.today(), *args, **kwargs):
+        super(WardNightSpider, self).__init__(*args, **kwargs)
+        self.start_urls = [spreadsheet_url + '/values/A3:N100?key=' + google_api_key]
+        self.start_date = start_date
```
[the remark this answers is not in the document]
Comment (jim) [reply documenters_aggregator/spiders/ward_night.py]: I think I can clean this up. At some point I was thinking we'd need to pass those values in, but at this point they are always ENV variables and it now seems unnecessary.

**Pull request merged**